commercetools-models/src/test/java/io/sphere/sdk/carts/queries/CartQueryIntegrationTest.java

```diff
@@ -59,6 +63,46 @@ public void expandDiscountCodeReference() throws Exception {
     }
 
     @Test
+    public void expandLineItemsDiscount() throws Exception {
+        final RelativeCartDiscountValue relativeCartDiscountValue = RelativeCartDiscountValue.of(15000);
+        withDiscountedLineItem(client(), relativeCartDiscountValue, (cart) -> {
+            final CartQuery query = CartQuery.of()
+                    .withPredicates(m -> m.id().is(cart.getId()))
+                    .withExpansionPaths(m -> m.lineItems().discountedPricePerQuantity().discountedPrice().includedDiscounts().discount());
+            assertEventually(() -> {
+                final Cart loadedCart = client().executeBlocking(query).head().get();
+                final Reference<CartDiscount> cartDiscountReference =
+                        loadedCart.getLineItems().get(0).getDiscountedPricePerQuantity().get(0).getDiscountedPrice().getIncludedDiscounts().get(0).getDiscount();
+                assertThat(cartDiscountReference.getObj()).isNotNull();
+                final CartDiscount lineItemDiscount = cartDiscountReference.getObj();
+                assertThat(lineItemDiscount).isNotNull();
```

Comment:
I think you can omit this check, since you have already `assertThat(cartDiscountReference.getObj()).isNotNull();`